shared/general/labels.md

```diff
@@ -1,15 +1,18 @@
-Label | Default | Description | Valid from
---- | --- | --- | ---
-io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container. | v7.22.0
-io.{{ $names.company.short }}.features.dbus | false | Bind mounts the host OS dbus into the container using `/run/dbus:/host/run/dbus`. | v7.22.0
-io.{{ $names.company.short }}.features.sysfs | false | Bind mounts the host OS `/sys` into the container. | v10.8.0
-io.{{ $names.company.short }}.features.procfs | false | Bind mounts the host OS `/proc` into the container. | v10.8.0
-io.{{ $names.company.short }}.features.kernel-modules | false | Bind mounts the host OS `/lib/modules` into the container (i.e. `/lib/modules:/lib/modules`). | v7.22.0
-io.{{ $names.company.short }}.features.firmware | false | Bind mounts the host OS `/lib/firmware` into the container. | v7.22.0
-io.{{ $names.company.short }}.features.supervisor-api | false | Ensures that `{{ $names.company.allCaps }}_SUPERVISOR_HOST`, `{{ $names.company.allCaps }}_SUPERVISOR_PORT`, `{{ $names.company.allCaps }}_SUPERVISOR_ADDRESS`, and `{{ $names.company.allCaps }}_SUPERVISOR_API_KEY` are added to the container environment variables, so the supervisor API can be used. | v7.22.0
-io.{{ $names.company.short }}.features.{{ $names.company.short }}-api | false | When enabled, it will make sure that `{{ $names.company.allCaps }}_API_KEY` is added to the container environment variables. | v7.22.0
-io.{{ $names.company.short }}.update.strategy | download-then-kill | Set the application [update strategy][update-strategy]. | v7.22.0
-io.{{ $names.company.short }}.update.handover-timeout | 60000 | Time, in milliseconds, before an old container is automatically killed. Only used with the `hand-over` [update strategy][hand-over]. | v7.22.0
+Label | Default | Description | Supervisor | balenaOS*
+--- | --- | --- | --- | ---
+io.{{ $names.company.short }}.features.balena-socket | false | Bind mounts the balena container engine socket into the container. | v7.23.0 | v2.21.0
```

Comment:
I have also changed v7.22.0 to v7.23.0 solely because of what I read in the supervisor changelog:
https://github.com/balena-io/balena-supervisor/blob/master/CHANGELOG.md
```
v7.23.0 - 2018-10-18
• Replace io.resin labels (and their env vars) with io.balena equivalents [Pablo Carranza Velez]
...
```
Which is why I've also added @20k-ultra, @richbayliss and @CameronDiver as reviewers. :-)
In any case, supervisor versions 7.22.0 and 7.23.0 are not separately shipped in balenaOS versions, so the balenaOS version column would have the same value either way.